{% extends "layout.html" %}

{% block head %}<style>
  .bodyParagraph h5 {
    margin-left: 0;
  }
</style>{% endblock %}
  
{% block title%}Engineering Success{% endblock %}

{% block img0url %}https://static.igem.wiki/teams/4190/wiki/banner-gif/ezgif-1-1019c9da57.gif{% endblock %}
{% block img0desc %}{% endblock %}

{% block header %}

    <div class="carousel-inner" id ="engineering_header">
      <div class="carousel-item active">
        <img src="{{ self.img0url() }}" class="d-block" alt="{{ self.img0desc() }}">
        <div class="carousel-caption">
          <h1>{{ self.title() }}</h1>
          <h4>{{ self.img0desc() }}</h4>
        </div>
      </div>
    </div>

{% endblock %}

{% block sideBarEntries %}

  <a class="nav-link active" aria-current="page" href="#engineering1">Ex-4 in <i>E. coli</i></a>
  <a class="nav-link" href="#engineering2">Ex-4 in <i>S. cerevisiae</i></a>
  <a class="nav-link" href="#engineering3">Producing GLP-1 R in <i>E. coli</i></a>
  <a class="nav-link" href="#engineering4">Binding Affinity Assay</a>
  <a class="nav-link" href="#engineering5">Modeling</a>
  <a class="nav-link" href="#Citation-Accordion">Citations</a>

{% endblock %}

{% block page_content %}
    <h4 id="engineering1" class="sectionHead"><strong>Ex-4 in <i>E. coli</i></strong></h4>
      <div class="bodyParagraph">
      <div id="Ecarousel1" class="carousel slide carousel-dark engineeringCarousel" data-bs-ride="carousel" data-bs-interval="false">
        <div class="carousel-indicators">
        <button type="button" data-bs-target="#Ecarousel1" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
        <button type="button" data-bs-target="#Ecarousel1" data-bs-slide-to="1" aria-label="Slide 2"></button>
        <button type="button" data-bs-target="#Ecarousel1" data-bs-slide-to="2" aria-label="Slide 3"></button>
        <button type="button" data-bs-target="#Ecarousel1" data-bs-slide-to="3" aria-label="Slide 4"></button>
        <button type="button" data-bs-target="#Ecarousel1" data-bs-slide-to="4" aria-label="Slide 5"></button>
      </div>
        <div class="carousel-inner">
          <div class="carousel-item active" data-bs-interval="false">
            <h5><strong>First Iteration</strong></h5>
            <p><strong>Design:</strong> We derived our backbone from the pET28:GFP <a class="glossaryRef" title="Plasmid" href="https://2022.igem.wiki/ucsc/glossary#Plasmid">plasmid</a> via inverse <a class="glossaryRef" title="PCR" href="https://2022.igem.wiki/ucsc/glossary#PCR">PCR</a> [<a class="citation" href="#Citation-Accordion">1</a>]. pET28:GFP contains a prokaryotic origin of replication, kanamycin antibiotic resistance, green fluorescent protein (GFP), a T7 <a class="glossaryRef" title="Promoter" href="https://2022.igem.wiki/ucsc/glossary#Promoter">promoter</a> and a T7 terminator. The <a class="glossaryRef" title="E_coli" href="https://2022.igem.wiki/ucsc/glossary#E_coli"><i>E. coli</a></i> codon optimized <a class="glossaryRef" title="Ex-4" href="https://2022.igem.wiki/ucsc/glossary#Ex-4">Ex-4</a> sequence was inserted into the space occupied by GFP on the pET28:GFP plasmid. Our Ex-4 gene insert includes: BsaI restriction enzyme site, <a class="glossaryRef" title="Shine-Dalgarno" href="https://2022.igem.wiki/ucsc/glossary#Shine-Dalgarno">Shine-Dalgarno</a> sequence, <a class="glossaryRef" title="His-Tag" href="https://2022.igem.wiki/ucsc/glossary#His-Tag">His-Tag</a>, linker sequence, <a class="glossaryRef" title="Enterokinase" href="https://2022.igem.wiki/ucsc/glossary#Enterokinase">enterokinase</a> <a class="glossaryRef" title="Cut-site" href="https://2022.igem.wiki/ucsc/glossary#Cut-site">cut site</a>, and the Ex-4 codon optimized sequence. To learn more about each of the components of our Ex-4 gene insert, check out our <a href="{{ url_for('pages', page='contribution') }}">Contributions</a> page!</p>
            <br>

          </div>
          <div class="carousel-item" data-bs-interval="false">
            
            <p><strong>Build:</strong> Golden Gate Assembly (<a class="glossaryRef" title="GGA" href="https://2022.igem.wiki/ucsc/glossary#GGA">GGA</a>) was used to insert Ex-4 into the plasmid backbone. GGA is a modular assembly method by which multiple DNA fragments can be simultaneously integrated into a plasmid backbone [<a class="citation" href="#Citation-Accordion">2</a>]. GGA exploits type IIS restriction enzymes, which cleave DNA surrounding
the recognition site, so the final <a class="glossaryRef" title="Recombination" href="https://2022.igem.wiki/ucsc/glossary#Recombination">recombinant</a> product has no indication of integration. All type II restriction enzymes leave <a class="glossaryRef" title="Sticky-Ends" href="https://2022.igem.wiki/ucsc/glossary#Sticky-Ends">sticky ends</a> on the backbone and the gene insert. These sticky ends bind to one another since they are designed to have complementary nucleotide sequences. Specifically, type IIS restriction enzymes cut downstream of the binding site by a certain number of nonspecific nucleotides. This number varies based on the type IIS restriction enzyme used. This means the sticky ends can be engineered to seamlessly integrate DNA without inserting residual nucleotides from the restriction enzyme binding site. This is deemed “scarless integration”, and prevents <a class="glossaryRef" title="Mutagenesis" href="https://2022.igem.wiki/ucsc/glossary#Mutagenesis">mutagenesis</a> during transcription of the amino acid sequences. The deletion of restriction enzyme sites also means digestion and ligation can be accomplished simultaneously.</p>
            <br>

          </div>
          <div class="carousel-item" data-bs-interval="false">
            
             <p><strong>Test:</strong> Once Ex-4 containing plasmids were constructed via a one step digestion and ligation GGA, they were transformed into <a class="glossaryRef" title="DH10B" href="https://2022.igem.wiki/ucsc/glossary#DH10B">DH10B</a> <i>E. coli</i>. No colonies grew after this first GGA attempt.</p>
             <br>

          </div>
      
          <div class="carousel-item" data-bs-interval="false">
            <h5><strong>Second Iteration</strong></h5>
            <p><strong>Design & Build:</strong> Consequently, we decided to try a longer thermocycler protocol that cycles through digestion and ligation steps to increase the probability of successful plasmid construction. For this next attempt using GGA, we pursued both the shorter and longer <a class="citation" style="display: inline; "href="{{ url_for('pages', page='protocols') }}">protocols</a>.</p>
            <br>
          </div>
          <div class="carousel-item" data-bs-interval="false">
            <p><strong>Test & Learn:</strong> These plasmids were again transformed into DH10B <i>E. coli</i> and incubated overnight. Colonies grew after this transformation, implying the longer thermocycler protocol may have contributed to a higher transformation efficiency. After testing both settings, we decided to use the longer thermocycler setting during GGA for constructing our <a class="glossaryRef" title="pET28:Ex-4" href="https://2022.igem.wiki/ucsc/glossary#pET28:Ex-4">pET28:Ex-4</a> plasmid. After we performed <a class="glossaryRef" title="Colony-PCR" href="https://2022.igem.wiki/ucsc/glossary#Colony-PCR">colony PCR</a> on the DH10B transformants and performed gel electrophoresis, we sent samples with bands at the expected size to sequencing. Of the seven samples sent to sequencing, only one contained a mutation. Plasmid samples with our Ex-4 inserts displaying no mutations in sequencing results were then transformed into the protein producing <i>E. coli</i> <a class="glossaryRef" title="BL21" href="https://2022.igem.wiki/ucsc/glossary#BL21">BL21</a> and <a class="glossaryRef" title="C41" href="https://2022.igem.wiki/ucsc/glossary#C41">C41</a>.</p>
            <br>
          </div>
  <button class="engineeringButton carousel-control-prev" type="button" data-bs-target="#Ecarousel1" data-bs-slide="prev">
    <span class="carousel-control-prev-icon" aria-hidden="true"></span>
    <span class="visually-hidden">Previous</span>
  </button>
  <button class="engineeringButton carousel-control-next" type="button" data-bs-target="#Ecarousel1" data-bs-slide="next">
    <span class="carousel-control-next-icon" aria-hidden="true"></span>
    <span class="visually-hidden">Next</span>
  </button>
</div>
</div>
</div>

    <br> 

    <h4 id="engineering2" class="sectionHead"><strong>Ex-4 in <i>S. cerevisiae</i></strong></h4>
      <div class="bodyParagraph">
      <div id="Ecarousel2" class="carousel slide carousel-dark engineeringCarousel engineeringCarousel" data-bs-ride="carousel" data-bs-interval="false">
        <div class="carousel-indicators">
        <button type="button" data-bs-target="#Ecarousel2" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
        <button type="button" data-bs-target="#Ecarousel2" data-bs-slide-to="1" aria-label="Slide 2"></button>
        <button type="button" data-bs-target="#Ecarousel2" data-bs-slide-to="2" aria-label="Slide 3"></button>
        <button type="button" data-bs-target="#Ecarousel2" data-bs-slide-to="3" aria-label="Slide 4"></button>
      </div>
        <div class="carousel-inner">
          <div class="carousel-item active" data-bs-interval="false">
            <h5><strong>First Iteration</strong></h5>
            <p><strong>Design:</strong> For the construction of our <i>S. cerevisiae</i> genomic insert, we used a <a class="glossaryRef" title="GAL1-promoter" href="https://2022.igem.wiki/ucsc/glossary#GAL1-promoter">GAL1 promoter</a> and a <a class="glossaryRef" title="PGK1" href="https://2022.igem.wiki/ucsc/glossary#PGK1">PGK1</a> terminator. The GAL1 promoter and PGK1 terminator sequences are derived from the yeast Modular Cloning (<a class="glossaryRef" title="MoClo" href="https://2022.igem.wiki/ucsc/glossary#MoClo">MoClo</a>) library [<a class="citation" href="#Citation-Accordion">3</a>]. Protein production is initiated by the GAL1 promoter when <i>S. cerevisiae</i> is grown up in a galactose based liquid media.</p> 
            <p>We used the plasmid backbones <a class="glossaryRef" title="mg-int-leu2-kan-1420" href="https://2022.igem.wiki/ucsc/glossary#mg-int-leu2-kan-1420">mg-int-leu2-kan-1420</a> (hereafter referred to as <a class="glossaryRef" title="pLEU" href="https://2022.igem.wiki/ucsc/glossary#pLEU">pLEU</a>) and <a class="glossaryRef" title="mg-int-trp1-hyg-1432-4a" href="https://2022.igem.wiki/ucsc/glossary#mg-int-trp1-hyg-1432-4a">mg-int-trp1-hyg-1432-4a</a> (hereafter referred to as <a class="glossaryRef" title="pTRP" href="https://2022.igem.wiki/ucsc/glossary#pTRP">pTRP</a>) for eventual genomic integration of our Ex-4 gene insert. pLEU has an origin of replication for <i>E. coli</i>, a yeast <a class="glossaryRef" title="G418" href="https://2022.igem.wiki/ucsc/glossary#G418">G418</a> resistance as a selection marker and <a class="glossaryRef" title="LEU2" href="https://2022.igem.wiki/ucsc/glossary#LEU2">LEU2</a> homology sites for integration into the host genome. pTRP is identical to the previous plasmid, except that it has hygromycin resistance in place of G418 resistance and <a class="glossaryRef" title="TRP1" href="https://2022.igem.wiki/ucsc/glossary#TRP1">TRP1</a> homology sites in place of LEU2. This means that this plasmid is integrated at a different spot inside the genome. Our insert was placed in between the two BsaI cut sites [<a class="citation" href="#Citation-Accordion">4</a>] already present on both the plasmid backbones. Because the promoter and terminator were cut out by our restriction enzyme digest, we added these parts as well during GGA.</p>
             <p>The <i>S. cerevisiae</i> Ex-4 sequence was optimized through IDT [<a class="citation" href="#Citation-Accordion">5</a>]. An elastin-based polypeptide half-life extender sequence was added to our insert on the C-terminus. The half-life extender is composed of a ten-fold repeat of the VPGVG amino acid sequence, and increases the half-life of Ex-4 by 3.7 times [<a class="citation" href="#Citation-Accordion">6</a>]. We hypothesize this may allow for increased <a class="glossaryRef" title="Bioavailability" href="https://2022.igem.wiki/ucsc/glossary#Bioavailability">bioavailability</a> of orally administered Ex-4. To learn more about each of the components of the <i>S. cerevisiae</i> genomic insert, check out our  <a href="{{ url_for('pages', page='contribution') }}">Contributions</a> page!</p></p>
            <br>
          </div>
          <div class="carousel-item" data-bs-interval="false">
            <p><strong>Build:</strong> Our genomic insert for <i>S. cerevisiae</i> was assembled using the yeast MoClo Library [<a class="citation" href="#Citation-Accordion">3</a>]. This pivotal research in yeast enables rapid design of plasmids for protein expression. The MoClo library provides resources for easy, one-step constructions of yeast plasmids through GGA. Predesigned plasmid parts can be interchanged quickly, allowing for easy testing of multiple plasmid constructs. </p>
     <p>The plasmid backbones, GAL1 promoter, Ex-4 insert, and PGK1 terminator were assembled simultaneously through a one tube GGA reaction. These customized plasmids were subsequently grown in DH10B <i>E. coli</i> post GGA and then introduced into our yeast strains. We transformed <a class="glossaryRef" title="pLEU:Ex-4" href="https://2022.igem.wiki/ucsc/glossary#pLEU:Ex-4">pLEU:Ex-4</a> into <i>S. cerevisiae</i> strain number <a class="glossaryRef" title="ROY-6783" href="https://2022.igem.wiki/ucsc/glossary#ROY-6783">ROY 6783</a>. ROY 6783 has a wild type LEU2 gene in the chromosome, allowing integration and <a class="glossaryRef" title="Auxotrophic" href="https://2022.igem.wiki/ucsc/glossary#Auxotrophic">auxotrophic</a> selection within the leucine production site of the genome. For the <a class="glossaryRef" title="pTRP:Ex-4" href="https://2022.igem.wiki/ucsc/glossary#pTRP:Ex-4">pTRP:Ex-4</a> plasmid, we utilized <i>S. cerevisiae</i> strain number <a class="glossaryRef" title="ROY-4513" href="https://2022.igem.wiki/ucsc/glossary#ROY-4513">ROY 4513</a>. ROY 4513 has a wild type TRP1 gene in the chromosome, allowing integration and auxotrophic selection within the tryptophan production site of the genome.</p> 
            <br>
          </div>
          <div class="carousel-item" data-bs-interval="false">
            <p><strong>Testing:</strong> We performed transformation of <i>S. cerevisiae</i> with the linearized integrative plasmids purified from our DH10B <i>E. coli</i> colonies. An additional linear plasmid was utilized in the pLEU:Ex4 transformation culture. This plasmid contained the  <a class="glossaryRef" title="CRISPR-Cas9" href="https://2022.igem.wiki/ucsc/glossary#CRISPR-Cas9">CRISPR-Cas9</a> protein coding gene and a single guide RNA targeting the LEU2 integration sites. The CRISPR-Cas9 protein cleaves and creates a chromosome break on the integration sites, which promotes <a class="glossaryRef" title="Homologous-Repair" href="https://2022.igem.wiki/ucsc/glossary#Homologous-Repair">homologous repair</a> and increases the efficiency of integration of the pLEU plasmid.</p>
     <p>The genomic DNA samples from pLEU:Ex4 and pTRP:Ex4 transformants were used for PCR reactions to check for gene insert integration via amplification. For each template DNA, the forward primer on the GAL1 promoter and the reverse primer on the PGK1 terminator were used to amplify the complete sequence of Ex-4. Bands at the expected size confirmed the presence of the desired protein coding sequence. Another set of primer pairs designed around the LEU2 <a class="glossaryRef" title="Homology-region" href="https://2022.igem.wiki/ucsc/glossary#Homology-region">homology region</a> and TRP1 homology region were run in separate PCR reaction mixtures to amplify the wildtype and integrated sequence of LEU2 and TRP1 integration sites. Lastly, the primer pair used previously to confirm GGA before the <i>S. cerevisiae</i> transformation step was used as a positive control.</p>
     <p>PCR products of the Ex-4 <a class="glossaryRef" title="Amplicon" href="https://2022.igem.wiki/ucsc/glossary#Amplicon">amplicon</a> from the <i>S. cerevisiae</i> genome were prepared and sent for single primer extension <a class="glossaryRef" title="Sanger-Sequencing" href="https://2022.igem.wiki/ucsc/glossary#Sanger-Sequencing">Sanger Sequencing</a>. The same forward primer and reverse primer used for PCR reactions were used for separate sequencing on the forward and reverse strands of each sample. Lastly, we lysed our cells to extract Ex-4 and purified it through immobilized metal affinity chromatography (<a class="glossaryRef" title="IMAC" href="https://2022.igem.wiki/ucsc/glossary#IMAC">IMAC</a>). We ran our purified protein on an <a class="glossaryRef" title="SDS-PAGE" href="https://2022.igem.wiki/ucsc/glossary#SDS-PAGE">SDS-PAGE</a> to see if there was a His-tag protein at the expected size of our Ex-4 protein.</p>
          <br>
          </div>
          <div class="carousel-item" data-bs-interval="false">
            <p><strong>Learn:</strong> Our genomic DNA gel electrophoresis showed bands at the expected nucleotide size for all sets of primers at both the LEU2 and TRP1 sites. To confirm that these nucleotide segments matched what we digitally designed, we sent our PCR products for Sanger Sequencing. We observed a 100% match on the Ex-4 protein coding region from all four sequencing results. Our SDS-PAGE results further confirmed that there was a His-tag protein of the appropriate size being produced by our yeast. Our work with <i>S. cerevisiae</i> also taught our team how to successfully perform a genomic integration of DNA into <i>>S. cerevisiae</i>.</p>
            <br>
          </div>
  <button class="engineeringButton carousel-control-prev" type="button" data-bs-target="#Ecarousel2" data-bs-slide="prev">
    <span class="carousel-control-prev-icon" aria-hidden="true"></span>
    <span class="visually-hidden">Previous</span>
  </button>
  <button class="engineeringButton carousel-control-next" type="button" data-bs-target="#Ecarousel2" data-bs-slide="next">
    <span class="carousel-control-next-icon" aria-hidden="true"></span>
    <span class="visually-hidden">Next</span>
  </button>
</div>
</div>
</div>

   <br>

  <h4 id="engineering3" class="sectionHead"><strong>GLP-1 R in <i>E. coli</i></strong></h4>
      <div class="bodyParagraph">
      <div id="Ecarousel3" class="carousel slide carousel-dark engineeringCarousel engineeringCarousel" data-bs-ride="carousel" data-bs-interval="false">
        <div class="carousel-indicators">
        <button type="button" data-bs-target="#Ecarousel3" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
        <button type="button" data-bs-target="#Ecarousel3" data-bs-slide-to="1" aria-label="Slide 2"></button>
        <button type="button" data-bs-target="#Ecarousel3" data-bs-slide-to="2" aria-label="Slide 3"></button>
        <button type="button" data-bs-target="#Ecarousel3" data-bs-slide-to="3" aria-label="Slide 4"></button>
        <button type="button" data-bs-target="#Ecarousel3" data-bs-slide-to="4" aria-label="Slide 5"></button>
        <button type="button" data-bs-target="#Ecarousel3" data-bs-slide-to="5" aria-label="Slide 6"></button>
        <button type="button" data-bs-target="#Ecarousel3" data-bs-slide-to="6" aria-label="Slide 7"></button>
        <button type="button" data-bs-target="#Ecarousel3" data-bs-slide-to="7" aria-label="Slide 8"></button>
        <button type="button" data-bs-target="#Ecarousel3" data-bs-slide-to="8" aria-label="Slide 9"></button>
        <button type="button" data-bs-target="#Ecarousel3" data-bs-slide-to="9" aria-label="Slide 10"></button>
      </div>
        <div class="carousel-inner">
          <div class="carousel-item active" data-bs-interval="false">
            <h5><strong>First Iteration</strong></h5>
            <p><strong>Design:</strong> A codon optimized sequence for the GLP-1 <a class="glossaryRef" title="Receptor" href="https://2022.igem.wiki/ucsc/glossary#Receptor">receptor</a> (<a class="glossaryRef" title="GLP-1-R" href="https://2022.igem.wiki/ucsc/glossary#GLP-1-R">GLP-1 R</a>) was designed for <i>E. coli</i> to test the <a class="glossaryRef" title="Binding-Affinity" href="https://2022.igem.wiki/ucsc/glossary#Binding-Affinity">binding affinity</a> of our Ex-4 in both <i>E. coli</i> and <i>S. cerevisiae</i>. This sequence was inserted into the same pET28:GFP backbone as the Ex-4 insert for <i>E. coli</i>. The gene insert for the GLP-1 R includes: BsaI restriction enzyme site, Shine-Dalgarno sequence, His-tag, linker sequence, <a class="glossaryRef" title="TEV" href="https://2022.igem.wiki/ucsc/glossary#TEV">TEV</a> cut site, and GLP-1 R codon optimized sequence. The insert includes a TEV cut site instead of an enterokinase cut site, but both have equivalent proteolytic functions. We inserted an alanine in between the TEV cut site and the GLP-1 R sequence because <a class="glossaryRef" title="TEV-protease" href="https://2022.igem.wiki/ucsc/glossary#TEV-protease">TEV protease</a> will only cut before serine, glycine, alanine, methionine, cysteine, or histidine amino acids [<a class="citation" href="#Citation-Accordion">7</a>]. To learn more about each of the components of the GLP-1 R gene insert, check out our  <a href="{{ url_for('pages', page='contribution') }}">Contributions</a> page!</p>
          <br>
          </div> 
          <div class="carousel-item" data-bs-interval="false">
            
            <p><strong>Build:</strong> The GLP-1 R was inserted into the pET28:GFP backbone through GGA. Initial assembly protocol suggested a 1:1 ratio of insert to backbone. We transformed our GGA assembled plasmids into DH10B <i>E. coli</i>, and incubated overnight with the expectation of seeing colonies the next day. To confirm the colonies we grew contained successfully ligated GGA plasmids, we performed colony PCR and gel electrophoresis. The GLP-1 R insert is 505 base pairs long, so using a 10kb ladder we were expecting bands to show up near the bottom of the gel.</p>
            <br>
          </div> 
          <div class="carousel-item" data-bs-interval="false">
            
            <p><strong>Test & Learn:</strong> Our gel electrophoresis had bands well below the expected position, indicating that our GGA was not successful in correctly assembling the GLP-1 R plasmid. The bands traveled far enough down the gel that they seemed to be 100 base pairs long. We hypothesized that our PCR primers ligated together, creating <a class="glossaryRef" title="Primer dimers" href="https://2022.igem.wiki/ucsc/glossary#Primer dimers">primer dimers</a>, and the GLP-1 R sequence was not successfully inserted into the pET28:GFP backbone.</p>
            <br>
          </div>
          <div class="carousel-item" data-bs-interval="false">
            <h5><strong>Second Iteration</strong></h5>
            <p><strong>Design:</strong> To test this hypothesis, we performed GGA with a 2:1 ratio of insert to backbone with the hope that increasing the number of GLP-1 R inserts would increase the probability of successful plasmid construction.</p>
            <br>
          </div>
          <div class="carousel-item" data-bs-interval="false">
              <p><strong>Build & Test:</strong> We transformed these GGA reactions into DH10B <i>E. coli</i> as we had done previously, and let them grow overnight. We performed colony PCR and gel electrophoresis to determine if the new GGA reactions were successful.</p>
            <br>
          </div>
          <div class="carousel-item" data-bs-interval="false">
              <p><strong>Learn:</strong> The gel electrophoresis showed the same bands as the previous iteration; the GGA primers formed primer dimers and the pET28:GFP backbone annealed to itself. We then decided to send our GLP-1 R insert and pET28:GFP backbone to sequencing to determine if the ends of the backbone or insert degraded, thus preventing proper plasmid assembly. The sequencing results for the backbone indicated that our backbone did not degrade, however, the sequencing results for the insert were inconclusive.</p>
            <br>
          </div>
          <div class="carousel-item" data-bs-interval="false">
            <h5><strong>Third Iteration</strong></h5>
            <p><strong>Design:</strong> We decided to try 2:1 GGA again, but this time, use fresh reagents and amplify our GLP-1 R insert to ensure the GLP-1 R insert's ends are correct. </p>
            <br>
          </div>
          <div class="carousel-item" data-bs-interval="false">
              <p><strong>Build:</strong> We performed PCR to amplify our GLP-1 R insert and used this amplified insert in the 2:1 GGA reaction. These GGA reactions were transformed into DH10B <i>E. coli</i> and incubated overnight. We performed colony PCR and gel electrophoresis as we had done in the last iteration.</p>
            <br>
          </div>
          <div class="carousel-item" data-bs-interval="false">
              <p><strong>Test:</strong> For the first time, we saw bands at the expected position of around 500 bases! We excitedly performed miniprep on those colonies to extract the plasmid and sent them to sequencing to further confirm that the plasmid was constructed correctly. Sequencing results concluded that we properly assembled the GLP-1 R insert into the pET28:GFP backbone. These iterations taught us that GGA is finicky and can fail in multiple ways. Our success came from improving upon multiple iterations of the engineering cycle. Our next steps were then transforming into C41 <i>E. coli</i>, and performing IMAC and SDS-PAGE.</p>
            <br>
          </div>
          <div class="carousel-item" data-bs-interval="false">
              <p><strong>Learn:</strong> The SDS-PAGE had a band in the pellet at 17<a class="glossaryRef" title="kD" href="https://2022.igem.wiki/ucsc/glossary#kD">kD</a>, the expected weight of the GLP-1 R. This suggests that the receptor is insoluble in <i>E. coli</i>. Our next steps include refolding the receptor in <i>E. coli</i>, so it will bind correctly to our Ex-4.</p>
            <br>
          </div>
  <button class="engineeringButton carousel-control-prev" type="button" data-bs-target="#Ecarousel3" data-bs-slide="prev">
    <span class="carousel-control-prev-icon" aria-hidden="true"></span>
    <span class="visually-hidden">Previous</span>
  </button>
  <button class="engineeringButton carousel-control-next" type="button" data-bs-target="#Ecarousel3" data-bs-slide="next">
    <span class="carousel-control-next-icon" aria-hidden="true"></span>
    <span class="visually-hidden">Next</span>
  </button>
</div>
</div>
</div>

<br>

    <h4 id="engineering4" class="sectionHead"><strong>Binding Affinity Assay</strong></h4>
      <div class="bodyParagraph">
      <div id="Ecarousel4" class="carousel slide carousel-dark engineeringCarousel engineeringCarousel" data-bs-ride="carousel" data-bs-interval="false">
        <div class="carousel-indicators">
        <button type="button" data-bs-target="#Ecarousel4" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
        <button type="button" data-bs-target="#Ecarousel4" data-bs-slide-to="1" aria-label="Slide 2"></button>
        <button type="button" data-bs-target="#Ecarousel4" data-bs-slide-to="2" aria-label="Slide 3"></button>
        <button type="button" data-bs-target="#Ecarousel4" data-bs-slide-to="3" aria-label="Slide 4"></button>
        <button type="button" data-bs-target="#Ecarousel4" data-bs-slide-to="4" aria-label="Slide 5"></button>
        <button type="button" data-bs-target="#Ecarousel4" data-bs-slide-to="5" aria-label="Slide 6"></button>
      </div>
        <div class="carousel-inner">
          <div class="carousel-item active" data-bs-interval="false">
            <h5><strong>First Iteration</strong></h5>
            <p><strong>Design:</strong> To test that our recombinant Ex-4 proteins function as expected, we planned an Octet binding affinity assay between Ex-4 and GLP-1 R. The <a class="glossaryRef" title="Octet-Machine" href="https://2022.igem.wiki/ucsc/glossary#Octet-Machine">Octet machine</a> determines the <a class="glossaryRef" title="Dissociation-Constant" href="https://2022.igem.wiki/ucsc/glossary#Dissociation-Constant">dissociation constant</a> (k<sub>d</sub>) of a protein-<a class="glossaryRef" title="Ligand" href="https://2022.igem.wiki/ucsc/glossary#Ligand">ligand</a> interaction using antibodies and a fiber optic biosensor [<a class="citation" href="#Citation-Accordion">8</a>]. First, this test requires the removal of one of the binding partner’s His-tag using a protease. Next, Penta-His antibodies on the biosensor tip attach to our His-tagged protein. Then, the tip submerges into the non-tagged protein sample, allowing binding reactions to occur. We decided to remove the <i>E. coli</i> Ex-4 His-tag and attach our purchased GLP-1 R on the Octet biosensor for our first round of testing.</p>
            <br>
          </div>
          <div class="carousel-item" data-bs-interval="false">
            
            <p><strong>Build:</strong> Contaminating salt content was removed from the chosen His-tagged Ex-4 samples using diafiltration with 3kD spin filters. After our Ex-4 was resuspended in Tris-HCL buffer, we incubated the protein with enterokinase overnight to cleave Ex-4’s His-tag. Batch purification isolated tag-free Ex-4, which was then analyzed via spectrophotometry at A280 to determine protein concentration.</p>
            <br>
          </div>
          <div class="carousel-item" data-bs-interval="false">
            
            <p><strong>Test & Learn:</strong> Protein concentration of our sample was around 0.001mg/mL, and did not indicate successful isolation of Ex-4. The 3kD spin filters we used during diafiltration may not have collected our 6.4kD Ex-4, as indicated by the DuBois lab who had similar issues isolating small proteins. </p>
            <br>
          </div>
      
          <div class="carousel-item" data-bs-interval="false">
            <h5><strong>Second Iteration</strong></h5>
            <p><strong>Design:</strong> Since a 1kD spin filter would have been difficult for our team to obtain, we hypothesized the His-tags may be kept on both binding partners. Ensuring the Octet biosensor was fully saturated in His-tagged GLP-1 R, and the Ex-4’s His-tag was shielded using a His-tag antibody, we believed an accurate binding reaction would still occur. </p>
            <br>
          </div>
          <div class="carousel-item" data-bs-interval="false">
            <p><strong>Build & Test:</strong> We performed the binding affinity assay using an Octet machine under the conditions described on the previous page. Controls for this assay included an unloaded sensor submerged into Ex-4 and a GLP-1 R loaded sensor submerged into His antibodies. </p> 
            <br>
          </div>
          <div class="carousel-item" data-bs-interval="false">
            <p><strong>Learn:</strong> The graph produced in real time from this assay indicated this experiment was not successful, suggesting binding did not occur. Our next step is to obtain a 1kD spin filter to properly capture Ex-4 so we can perform the binding affinity assay under the proper condition where only one binding partner has a His-tag.</p>
            <br>
          </div>
  <button class="engineeringButton carousel-control-prev" type="button" data-bs-target="#Ecarousel4" data-bs-slide="prev">
    <span class="carousel-control-prev-icon" aria-hidden="true"></span>
    <span class="visually-hidden">Previous</span>
  </button>
  <button class="engineeringButton carousel-control-next" type="button" data-bs-target="#Ecarousel4" data-bs-slide="next">
    <span class="carousel-control-next-icon" aria-hidden="true"></span>
    <span class="visually-hidden">Next</span>
  </button>
</div>
</div>
</div>

       <br> 

      <h4 id="engineering5" class="sectionHead"><strong>Modeling</strong></h4>
      <div class="bodyParagraph">
      <div id="Ecarousel5" class="carousel slide carousel-dark engineeringCarousel engineeringCarousel" data-bs-ride="carousel" data-bs-interval="false">
        <div class="carousel-indicators">
        <button type="button" data-bs-target="#Ecarousel5" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
        <button type="button" data-bs-target="#Ecarousel5" data-bs-slide-to="1" aria-label="Slide 2"></button>
        <button type="button" data-bs-target="#Ecarousel5" data-bs-slide-to="2" aria-label="Slide 3"></button>
      </div>
        <div class="carousel-inner">
          <div class="carousel-item active" data-bs-interval="false">
            <p><strong>Design:</strong> The first model we wrote predicted Ex-4's binding affinity in the presence of its endogenous competitor, <a class="glossaryRef" title="GLP-1" href="https://2022.igem.wiki/ucsc/glossary#GLP-1">GLP-1</a>, aptly named the competitive binding model. A crucial step in this process was deciding which formula best described our system, where both Ex-4 and GLP-1 generate the desired pharmacological effect. After speaking with Dr. Michael Stone, a biochemistry professor at UC Santa Cruz, we chose a variation of the Cheng-Prusoff equation [<a class="citation" href="#Citation-Accordion">7</a>] for its history of representing competitive binding interactions. For all parameters involved (e.g. IC<sub>50</sub>, K<sub>i</sub>), we found literature values by searching through online databases such as PubMed and Google Scholar.</p>
            <br>
          </div>
          <div class="carousel-item" data-bs-interval="false">
            <p><strong>Build and Test:</strong> We programmed the competitive binding model in Python 3 using Jupyter Notebook. While we initially attempted to use MATLAB, Python was ultimately the better choice due to our team's past experience with the language. We ran the code and adjusted our graph’s specifications for clarity of our model.</p>
          <br>
          </div>
      
          <div class="carousel-item" data-bs-interval="false">
            <p><strong>Learn:</strong> We learned this model does not appropriately represent our system because it does not account for the effect of GLP-1 and Ex-4 both producing insulin. The Cheng-Prusoff equation tells us the concentration at which Ex-4 inhibits 50% of the natural ligand binding. We changed our equation to <a class="glossaryRef" title="Fractional-Occupancy" href="https://2022.igem.wiki/ucsc/glossary#Fractional-Occupancy">fractional occupancy</a> because it tells us how many ligands are bound compared to the total amount of receptors in a system. This way, we would be able to more accurately compare the competitive binding between Ex-4 and GLP-1.</p>
          <br>
          </div>
  <button class="engineeringButton carousel-control-prev" type="button" data-bs-target="#Ecarousel5" data-bs-slide="prev">
    <span class="carousel-control-prev-icon" aria-hidden="true"></span>
    <span class="visually-hidden">Previous</span>
  </button>
  <button class="engineeringButton carousel-control-next" type="button" data-bs-target="#Ecarousel5" data-bs-slide="next">
    <span class="carousel-control-next-icon" aria-hidden="true"></span>
    <span class="visually-hidden">Next</span>
  </button>
</div>
</div>
</div>

<br>


 <div class="accordion bg-dark my-5" id="Citation-Accordion">
    <div class="accordion-item">
      <h2 class="accordion-header" id="Citation-AccordionHeading1">
        <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseCitations" aria-expanded="false" aria-controls="collapseCitations">
          Citations
        </button>
      </h2>
      <div id="collapseCitations" class="accordion-collapse collapse" aria-labelledby="Citation-AccordionHeading1" data-bs-parent="#Citation-Accordion">
        <div class="accordion-body">
          <ol class="noBP">
            <li id="citation1">[1]  D. L. Shis and M. R. Bennett, “Library of synthetic transcriptional AND gates built with split T7 RNA polymerase mutants,” <i>Proceedings of the National Academy of Sciences</i>, vol. 110, no. 13, pp. 5028–5033, Mar. 2013, doi: <a class="citation" href="https://www.pnas.org/doi/full/10.1073/pnas.1220157110">10.1073/pnas.1220157110</a>.</li>
              <li id="citation2">[2]  C. Engler, R. Gruetzner, R. Kandzia, and S. Marillonnet, “Golden Gate Shuffling: A One-Pot DNA Shuffling Method Based on Type IIs Restriction Enzymes,” <i>PLOS ONE</i>, vol. 4, no. 5, p. e5553, May 2009, doi: <a class="citation" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0005553">10.1371/journal.pone.00055533</a>.</li>
              <li id="citation3">[3]  M. E. Lee, W. C. DeLoache, B. Cervantes, and J. E. Dueber, “A Highly Characterized Yeast Toolkit for Modular, Multipart Assembly,” <i>ACS Synth. Biol</i>., vol. 4, no. 9, pp. 975–986, Sep. 2015, doi: : <a class="citation" href="https://pubs.acs.org/doi/10.1021/sb500366v">10.1021/sb500366v</a>.</li>
              <li id="citation4">[4]  “Frontiers | Golden Gate Cloning-Compatible DNA Replicon/2A-Mediated Polycistronic Vectors for Plants.” <a class="citation" href="https://www.frontiersin.org/articles/10.3389/fpls.2020.559365/full ">https://www.frontiersin.org/articles/10.3389/fpls.2020.559365/full </a>(accessed Oct. 05, 2022).</li>
              <li id="citation1">[5]  “Codon Optimization Tool | IDT,” <i>Integrated DNA Technologies.</i> <a class="citation" href="https://www.idtdna.com/pages/tools/codon-optimization-tool">https://www.idtdna.com/pages/tools/codon-optimization-tool</a>.</li>
              <li id="citation1">[6]  S. J. Jung <i>et al.</i>, “In-vivo half-life and hypoglycemic bioactivity of a fusion protein of exenatide and elastin-based polypeptide from recombinant Saccharomyces cerevisiae,” <i>Journal of Biotechnology</i>, vol. 303, pp. 16–24, Sep. 2019, doi: <a class="citation" href="https://doi.org/10.1016/j.jbiotec.2019.06.304"> 10.1016/j.jbiotec.2019.06.304</a>.</li>
              <li id="citation1">[7]   Y. Cheng and W. H. Prusoff, “Relationship between the inhibition constant (K1) and the concentration of inhibitor which causes 50 per cent inhibition (I50) of an enzymatic reaction,” <i>Biochem Pharmacol</i>, vol. 22, no. 23, pp. 3099–3108, Dec. 1973, doi: 10.1016/0006-2952(73)90196-2 <a class="citation" href="https://doi.org/10.1016/0006-2952(73)90196-2">10.1016/0006-2952(73)90196-2</a>.</li>
              <li id="citation1">[8]    “Octet® HIS1K Biosensors,” Sartorius. <a class="citation" href="https://www.sartorius.com/en/products/protein-analysis/octet-bli-detection/biosensors-chips-kits/antipenta-his-his1k-biosensors">https://www.sartorius.com/en
/products/protein-analysis/octet-bli-detection/biosensors-chips-kits/antipenta-his-his1k-biosensors </a> Accessed: Aug. 28, 2022.</li>
          </ol>
        </div>
      </div>
    </div>
  </div>

<br>

{% endblock %}

